Services/Interfaces/IBetableService.cs

```diff
@@ -3,7 +3,9 @@
     public interface IBetableService<T, UT>
```

Comment:
I don't get the purpose of this interface? 
each service should have it's own interface and expose only the methods needed